src/graphqlTypes.ts

```diff
@@ -117,7 +117,7 @@ export interface GQLResource extends GQLTaxonomyEntity {
   article?: GQLArticle;
   learningpath?: GQLLearningpath;
   relevanceId?: string;
-  resourceTypes?: Array<GQLResourceType | null>;
+  resourceTypes?: Array<GQLResourceType>;
   parentTopics?: Array<GQLTopic | null>;
   breadcrumbs?: Array<Array<string | null> | null>;
```

Comment:
Alle disse typene i ndla-frontend har blitt hentet fra graphql, nå som du har forandra de typene i graphql til å ikke inneholde `null` er det lurt å oppdatere alle typene i ndla-frontend til å ligne på det som ligger i graphql.

Reply:
Tenkte å vente med det inntil det blir godkjennt med GraphQL pr'en. Litt kjipt å gjøre den editen for å så måtte gjøre om på det.